src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -99,74 +108,89 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
+                }
+                perPageOptions={
+                  displayShowAllButton ? paginationDefaultValuesForm : paginationDefaultValuesModal
+                }
+                defaultToFullPage
+                isCompact={displayShowAllButton}
+                itemCount={data?.length}
+                page={pagination?.page}
+                perPage={pagination?.perPage}
+              />
+            </SplitItem>
+            {displayShowAllButton && <ShowAllBootableVolumesButton />}
+          </>
+        )}
       </Split>
-      <TableComposable variant={TableVariant.compact}>
-        <Thead>
-          <Tr>
-            {activeColumns.map((col, columnIndex) => (
-              <Th id={col?.id} key={col?.id} sort={getSortType(columnIndex)}>
-                {col?.title}
-              </Th>
+
+      {bootableVolumes && loaded ? (
+        <TableComposable variant={TableVariant.compact}>
+          <Thead>
+            <Tr>
+              {activeColumns.map((col, columnIndex) => (
+                <Th id={col?.id} key={col?.id} sort={getSortType(columnIndex)}>
+                  {col?.title}
+                </Th>
+              ))}
+            </Tr>
+          </Thead>
+          <Tbody>
+            {sortedData.map((bs) => (
+              <BootableVolumeRow
+                rowData={{
+                  bootableVolumeSelectedState: !displayShowAllButton
+                    ? selectedBootableVolumeState
+                    : [selectedBootableVolume, onSelectCreatedVolume],
+                  preference: preferencesMap[getLabel(bs, DEFAULT_PREFERENCE_LABEL)],
+                  pvcSource: getBootableVolumePVCSource(bs, pvcSources),
+                }}
+                activeColumnIDs={activeColumns?.map((col) => col?.id)}
+                bootableVolume={bs}
+                key={getName(bs)}
+              />
             ))}
-          </Tr>
-        </Thead>
-        <Tbody>
-          {sortedData?.map((bs) => (
-            <BootableVolumeRow
-              rowData={{
-                bootableVolumeSelectedState: !displayShowAllButton
-                  ? selectedBootableVolumeState
-                  : [selectedBootableVolume, onSelectCreatedVolume],
-                preference: preferencesMap[getLabel(bs, DEFAULT_PREFERENCE_LABEL)],
-                pvcSource: getBootableVolumePVCSource(bs, pvcSources),
-              }}
-              activeColumnIDs={activeColumns?.map((col) => col?.id)}
-              bootableVolume={bs}
-              key={getName(bs)}
-            />
-          ))}
-        </Tbody>
-      </TableComposable>
+          </Tbody>
+        </TableComposable>
+      ) : (
+        <EmptyState className="bootable-volume-list-empty-state">
+          <Trans ns="plugin__kubevirt-plugin" t={t}>
+            <Title headingLevel="h3">No volumes found</Title>
+            Click the <b> Add volume</b> button to add a volume to boot from
+          </Trans>
+        </EmptyState>
```

Comment:
@hstastna This empty state is no longer required for this version, you can remove it.
Please read last comment by UX in here: https://docs.google.com/document/d/1R2TY2fbXRxxWIikK-ssava27kI6s6tvEE0kJpSLsxM4/edit?disco=AAAAzAgx0g8

Reply:
Hmm, changes again? 🤦🏽